tests/integration/targets/snap/tasks/main.yml

```diff
@@ -4,20 +4,35 @@
 # and should not be used as examples of how to write Ansible roles #
 ####################################################################
 
-- name: install snapd
+- name: install snapd on ubuntu
   apt:
     name: snapd
     state: present
   register: snapd_install_ubuntu
```

Comment:
Can you use `notify:` and handlers to do the uninstall?

Reply:
Yes, I will change it in a couple of hours.